### DISCUSSION TITLE
`rails console`-equivalent tool for the epic stack?

### DISCUSSION THREAD

**Discussion by fullofcaffeine** [epicweb-dev/epic-stack#492] [Ideas]
Hi!

Coming from a Rails background, I grew very fond/used to `rails console`, which loads all models and allows you to have a CLI admin interface to fix stuff in the db/for customers when needed. I've found this related issue on the Prisma repo: https://github.com/prisma/prisma/issues/9563, and then this project https://github.com/egoist/prisma-repl, which looks like it's very similar (though untested).  In the context of shipping stuff fast, this is a super useful tool. Are there plans to add something like that to the epic stack? 

Cheers!

**Comment by kentcdodds** [comment]
This would be really cool. I'm pretty happy with prisma studio for stuff like this, but I can understand why someone would want a REPL.

This is definitely something that would be cool to have in the Epic Stack, but I don't personally have time to work on it. I welcome an example though!
